contrib/kubernetes/datahub/charts/datahub-frontend/templates/deployment.yaml

```diff
@@ -46,26 +46,38 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
             - name: http
-              containerPort: 9001
+              containerPort: 9002
```

Comment:
```suggestion
              containerPort: {{ .Values.service.port }}
```

Reply:
This is an internal port. I think it should be hardcoded. service.port field is used in the service.yml to define external port. WDYT?